tox.ini

```diff
@@ -105,10 +115,16 @@ commands =
 
 [testenv:asyncio-pinned]
 commands = {[testenv:asyncio]commands}
-deps = {[testenv:pinned]deps}
+deps = 
+    {[testenv:pinned]deps}
+    lxml==3.5.0
```

Comment:
After reverting the removal of `lxml==3.5.0` from `testenv:pinned`, this change should be reverted.

Reply:
`testenv:pinned` is only used by one job, and only add one dependencies. Wouldn't it make sense to remove it?